### ISSUE TITLE
Regression on vpc_source_addresses support

### ISSUE THREAD

**Issue by ifs-pauljegouic** [IBM-Cloud/terraform-provider-ibm#1530]
Hi there,

### Terraform Version

- 1.6
- 1.2.6

### Affected Resource(s)
Please list the resources as a list, for example:
- ibm_is_vpc

### Terraform Configuration Files
```hcl
data "ibm_is_vpc" "vpc" {
 name = "foobar"
}
```

### Expected Behavior

The `cse_addresses` are successfully retrieved.

### Actual Behavior

In 1.6, return empty list:

```json
{
    "classic_access" = false
    "crn" = "crn:v1:bluemix:public:is:eu-de:a/67a4395fde4e4e0ca52f5554b249ceb7::vpc:156515d7-4086-4534-b01a-93e0345626c2"
    "cse_source_addresses" = []
    "default_network_acl" = "1e511d05-a978-402c-bd5d-5ac95a058c1a"
    "id" = "156515d7-4086-4534-b01a-93e0345626c2"
    "name" = "vpc1"
    "resource_controller_url" = "https://cloud.ibm.com/vpc/network/vpcs"
    "resource_crn" = "crn:v1:bluemix:public:is:eu-de:a/67a4395fde4e4e0ca52f5554b249ceb7::vpc:156515d7-4086-4534-b01a-93e0345626c2"
    "resource_group" = "cab185d73a324dd6a69a1ae45e3844f7"
    "resource_group_name" = "cab185d73a324dd6a69a1ae45e3844f7"
    "resource_name" = "vpc1"
    "resource_status" = "available"
    "status" = "available"
    "subnets" = [
      {
        "available_ipv4_address_count" = 11
        "id" = "f91df4d0-56de-49c5-a3d3-e8e1efd6d052"
        "name" = "public-b"
        "status" = "available"
        "total_ipv4_address_count" = 16
      },
      {
        "available_ipv4_address_count" = 11
        "id" = "bc161adf-e41d-4d0c-9fc0-3d0ebee96215"
        "name" = "public-a"
        "status" = "available"
        "total_ipv4_address_count" = 16
      },
      {
        "available_ipv4_address_count" = 52
        "id" = "76ad8c75-ce7c-4de9-9232-9c5c34701a12"
        "name" = "private-b"
        "status" = "available"
        "total_ipv4_address_count" = 64
      },
      {
        "available_ipv4_address_count" = 47
        "id" = "895122e1-2592-41cc-843f-8632d6d6bc34"
        "name" = "private-a"
        "status" = "available"
        "total_ipv4_address_count" = 64
      },
    ]
    "tags" = []
  }
}
```

With 1.2.6, return complete list:

```json
{
  "activity_tracker_instance_id" = "crn:v1:bluemix:public:logdnaat:eu-de:a/67a4395fde4e4e0ca52f5554b249ceb7:8693b063-04d7-4c05-8851-003fe4570c2b::"
  "cis_id" = "crn:v1:bluemix:public:internet-svcs:global:a/67a4395fde4e4e0ca52f5554b249ceb7:f14511f9-4dd5-4a55-bd53-581c33c7c961::"
  "domain_id" = "ee214375da49ccb6d7b7f8ba196e9679:crn:v1:bluemix:public:internet-svcs:global:a/67a4395fde4e4e0ca52f5554b249ceb7:f14511f9-4dd5-4a55-bd53-581c33c7c961::"
  "logdna_instance_id" = "crn:v1:bluemix:public:logdna:eu-de:a/67a4395fde4e4e0ca52f5554b249ceb7:475f0d49-ee92-4e93-bbb1-fcb11a0b39cc::"
  "sysdig_instance_id" = "crn:v1:bluemix:public:sysdig-monitor:eu-de:a/67a4395fde4e4e0ca52f5554b249ceb7:542cd0f2-6cbd-435c-8e14-df5144fb5c1d::"
  "target_resource_group" = "681ed5c0507740ad9226a7d87cabbbfa"
  "vpc_id" = "156515d7-4086-4534-b01a-93e0345626c2"
  "vpc_source_addresses" = {
    "classic_access" = false
    "cse_source_addresses" = [
      {
        "address" = "10.249.165.91"
        "zone_name" = "eu-de-3"
      },
      {
        "address" = "10.249.160.20"
        "zone_name" = "eu-de-2"
      },
      {
        "address" = "10.249.159.196"
        "zone_name" = "eu-de-1"
      },
    ]
    "default_network_acl" = "1e511d05-a978-402c-bd5d-5ac95a058c1a"
    "id" = "156515d7-4086-4534-b01a-93e0345626c2"
    "name" = "vpc1"
    "resource_controller_url" = "https://cloud.ibm.com/vpc/network/vpcs"
    "resource_crn" = "crn:v1:bluemix:public:is:eu-de:a/67a4395fde4e4e0ca52f5554b249ceb7::vpc:156515d7-4086-4534-b01a-93e0345626c2"
    "resource_group" = "cab185d73a324dd6a69a1ae45e3844f7"
    "resource_group_name" = ""
    "resource_name" = "vpc1"
    "resource_status" = "available"
    "status" = "available"
  }
}
```

I have tested until version 1.5.0 and the regression was already present.

**Comment by hkantare** [comment]
Fixed in latest release https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.7.1

**Comment by ifs-pauljegouic** [comment]
Fixed with 1.7.1 

Thanks

**Comment by hkantare** [comment]
closing based on above comments
